.dockerignore

```diff
@@ -0,0 +1,3 @@
+*
+!docker/
+!pkg/docker
```

Comment:
Oh, there is an upstream `docker/` directory? Then please ignore it via this `.dockerignore`. If there was no `.dockerignore` before, then the upstream dockerfile will be broken anyway if they didn't ignore the rest.